lib/offline/background.ts

```diff
@@ -0,0 +1,21 @@
+import { jsonApiPost } from "../axios";
+import { processJournal, processTechnical } from "../process";
+import { offlineDB } from "./db";
+
+export async function submitOfflineJournals() {
+  const data = await offlineDB.farmerJournal.toArray();
+  for (const entry of data) {
+    console.log("post journal", entry.id);
+    await jsonApiPost("node/farmer_daily_journal", processJournal(entry.value))
+    await offlineDB.farmerJournal.delete(entry.id!);
```

Comment:
Do we only want to delete the entries if `jsonApiPost is successful?

Reply:
Yep, it should be
Otherwise, we have duplicates
jsonApiPost will throw an error if the response is not okay